rpcserver.go

```diff
@@ -1512,6 +1512,44 @@ func (r *rpcServer) ListChannels(ctx context.Context,
 			linkActive = link.EligibleToForward()
 		}
 
+		// Next, we'll determine whether we should add this channel to
+		// our list depending on the type of channels requested to us.
+		isActive := peerOnline && linkActive
+		isPublic := dbChannel.ChannelFlags&lnwire.FFAnnounceChannel ==
```

Comment:
This should be `isPublic := dbChannel.ChannelFlags&lnwire.FFAnnounceChannel != 0`